zilencer/management/commands/populate_db.py

```diff
@@ -197,6 +198,14 @@ def add_arguments(self, parser: CommandParser) -> None:
             "-n", "--num-messages", type=int, default=500, help="The number of messages to create."
         )
 
+        parser.add_argument(
+            "-d",
+            "--starting-n-days-ago",
```

Comment:
I suppose `--oldest-message-days` could be defensible?

Reply:
I think the reason I chose this name instead of something like that was because technically there might not be a message on the first day a message *could* be sent. But maybe it's not a big deal to slightly lie about that, because `--oldest-message-days` definitely sounds a lot better :)

Reply:
Yeah it seems fine to be a bit inaccurate in the name here :).